go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/giantswarm/apiextensions-application v0.6.1
-	github.com/giantswarm/clustertest v0.17.0
+	github.com/giantswarm/clustertest v0.17.1-0.20240418092820-fb25dad2e6c8
```

Comment:
Will update when https://github.com/giantswarm/clustertest/pull/166 is merged and released.